package.json

```diff
@@ -35,7 +35,7 @@
     "build": "babel src --out-dir bin --out-file-extension .mjs",
     "docs": "scripts/js-to-md.js src/*.js src/commands/*.js src/actions/*.js src/utils/*.js",
     "lint": "commitlint --from 23f6888 && eslint .",
-    "test": "mocha --recursive",
+    "test": "mocha --recursive --timeout 6000",
```

Comment:
Before I removed a global timeout in c9c8f9f1d8d3e89d636818e86beede5504960182. I propose to specify timeout explicitly in tests that needs them.